paddle/fluid/framework/CMakeLists.txt

```diff
@@ -165,9 +165,15 @@ cc_library(variable_helper SRCS variable_helper.cc DEPS lod_tensor)
 cc_library(naive_executor SRCS naive_executor.cc DEPS op_registry device_context scope framework_proto glog lod_rank_table feed_fetch_method graph_to_program_pass variable_helper)
 
 if(WITH_DISTRIBUTE)
-  cc_library(executor SRCS executor.cc DEPS op_registry device_context scope framework_proto glog lod_rank_table feed_fetch_method sendrecvop_grpc cares grpc++_unsecure grpc_unsecure gpr graph_to_program_pass variable_helper)
+    if(WITH_GRPC)
+  cc_library(executor SRCS executor.cc DEPS op_registry device_context scope framework_proto glog lod_rank_table feed_fetch_method sendrecvop_grpc cares grpc++_unsecure grpc_unsecure gpr ${GLOB_DISTRIBUTE_DEPS} graph_to_program_pass variable_helper)
```

Comment:
This if/else have many duplicated things, maybe SEND_RECV_OP_RPC is better.
```cmake
if (WITH_GRPC)
  set(SEND_RECV_OP_RPC sendrecvop_grpc)
else(WITH_BRPC)
  set(SEND_RECV_OP_RPCsendrecvop_brpc )
endif()
  
````

Reply:
Done. Thanks.